README.md

```diff
@@ -1,2 +1,44 @@
-# church-conference-capstone
-This is a project for an annual church conference. 
+# Project Name
+
+Faith Apostolic Annual Conference Website.
+
+![screenshot](./app_screenshot.png)
+
+This is a website that was created as a Microverse Capstone. The website is a mock of an annual conference of churches. There is a home page that consist of the main program and the speakers. The about page gives more information on the annual conference.
+
+## Built With
+
+- HTML
+- SASS
+- JavaScript
+
+## Live Demo
+
+[Live Demo Link](https://the-catalystmc.github.io/church-conference-capstone/)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps. You can either download the zip file, or clone the repository from [here](https://github.com/the-catalystmc/church-conference-capstone). After you can open the index.html file to load the website locally.
+
+## Authors
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/the-catalyst-mc)
+- Twitter: [@twitterhandle](https://twitter.com/catalystspeaks)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/rhyinestewart)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/the-catalystmc/church-conference-capstone/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+Original design idea by Cindy Shin in Behance
```

Comment:
[Optional]

- [ ] It will be better if you put a link to the Behance page of the author.